e2e/page-objects/patient/patient-detail/patient-allergy-dialog.po.ts

```diff
@@ -0,0 +1,43 @@
+import { by } from 'protractor';
+import { BasePage } from '../../base-page';
+import { Button, Datepicker, InputField } from '../../../widgets';
+
+
+export class PatientAllergyDetailPage extends BasePage {
+
+	constructor() {
+		super('patient-allergy-dialog');
+	}
+
+	public getSubmitButton(): Button {
+		return new Button(this.getObjectById('PatientSubmitButton'));
+	}
+
+	public getAllergyNotes(): InputField {
+		return new InputField(this.getObjectById('PatientAllergyNotes'));
+	}
+
+	public getAssertedDate(): Datepicker {
+		return new Datepicker(this.current.all(by.tagName('systelab-datepicker'))
+			.get(0));
+	}
+
+	public getLastOccurrenceDate(): Datepicker {
+		return new Datepicker(this.current.all(by.tagName('systelab-datepicker'))
+			.get(1));
+	}
+
+	public getAllergyCombobox() {
+		return this.current.element(by.tagName('allergy-combobox'));
+	}
+
+	public getAllergyComboInput() {
+		return this.getAllergyCombobox()
```

Comment:
A new widget must be created for the ComboBox with getInput() and getInputValue().

Reply:
Absolutely. I am on it.